policy/modules/admin/usbguard.fc

```diff
@@ -0,0 +1,9 @@
+/etc/usbguard						gen_context(system_u:object_r:usbguard_etc_t,s0)
+/etc/usbguard/rules\.conf				gen_context(system_u:object_r:usbguard_etc_t,s0)
+/etc/usbguard/usbguard-daemon\.conf			gen_context(system_u:object_r:usbguard_etc_t,s0)
+/etc/usbguard/IPCAccessControl\.d(/.*)?			gen_context(system_u:object_r:usbguard_etc_t,s0)
```

Comment:
These could probably be merged into one (regex) entry.

Reply:
Fixed in 36b7876 (updated while you reviewed).